.github/workflows/graphql-service-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: "CI - GraphQL Service"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - pstatus-graphql-ktor/**
+
+defaults:
+  run:
+    working-directory: pstatus-graphql-ktor/
+
+jobs:
+  unit-test:
+    name: Unit Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.22.0
+      - name: Vet
+        run: go vet ./...
+      - name: Run Tests
+        run: go test -coverprofile=c.out -coverpkg=./... ./...
+      - name: Show coverage
+        run: go tool cover -func=c.out
+      - name: Build Check
+        run: go build -o pstatus-graphql-ktor ./cmd/main.go
```

Comment:
The unit tests are not built in go, we use kotlin.  The run step should be something like `gradle test`.

Reply:
gradle test  step added